code/missionui/redalert.cpp

```diff
@@ -1055,6 +1099,38 @@ void red_alert_bash_wingman_status()
 	}
 }
 
+void red_alert_bash_wing_status()
+{
+	for (const auto& recorded_wing : Red_alert_wing_status){
+
+		for (int i = 0; i < MAX_WINGS; ++i){
+			auto& mission_wing = Wings[i];
+
+			// check that we have the same wing.  Then we check if the wings are actually compatible.
+			if (!stricmp(recorded_wing.wing_name.c_str(), mission_wing.name)){
+
+				// do we have sufficient waves in the current mission to handle where the wave should be from the previous mission?
+				if  ((mission_wing.num_waves < recorded_wing.current_wave) || (mission_wing.wave_count != recorded_wing.max_ships_per_wave)){
+					mprintf(("Red alert code cannot restore %s wing's info since there's a mismatch of total ships available (total waves and ships per wave). New ships for this wing will not appear.", mission_wing.name));
```

Comment:
This should be a warning
```suggestion
					Warning(LOCATION, "Red alert code cannot restore %s wing's info since there's a mismatch of total ships available (total waves and ships per wave). New ships for this wing will not appear.", mission_wing.name);
```